app/app.js

```diff
@@ -356,18 +369,25 @@ class MaterialColors {
       $('<div>')
           .addClass(this.CLASS_NAMES.colorTileValueName)
           .text(value.valueName.toUpperCase())
+          .click(() => {
+            let copyText = nunjucks.renderString(
+                this._CONFIG.value_copy_format || "--{{HUE}}-{{VALUE}}",
```

Comment:
i think it'll just be easier for developers to change this than designers... i don't feel super super strongly, but definitely prefer the simpler default :)